SwiftWisdom/Core/Foundation/Time/Date+Extensions.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Date+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Paul Rolfe on 1/28/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
```

Comment:
Test these. See DateExtensionTests in the original project, which shall rename nameless.